_assets/sass/elements/_footer.scss

```diff
@@ -7,7 +7,8 @@
 }
 
 .wrap-footer {
-    background: $black url('#{$image-path}/vi-shape7-tp.svg') no-repeat 10% center;
+    background-image: url('https://cdn.mitlibrary.net/files/branding/local/vi-shape7-tp.svg');
```

Comment:
Are we intentionally using the stage URLs?

Reply:
No, but that is a link to prod...I suspect there are others I missed and with check them more closely :)

Reply:
`https://cdn.libraries.mit.edu` is an alias, then? TIL...

Reply:
Oh... what were you using for prod? I didn't realize there was a different link that worked for prod...  Interesting. But yeah, the non-environment link is probably best to use in production.

Reply:
Yeah, `https://cdn.libraries.mit.edu` is what I'm using for Springshare. It seems to work. 🤷